odoo/addons/base/models/res_lang.py

```diff
@@ -1,20 +1,34 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+import base64
 import json
 import locale
 import logging
+import os
 import re
+import shutil
+import tarfile
+import tempfile
+from collections import defaultdict
+from datetime import timedelta
 from operator import itemgetter
+from io import BytesIO
+from urllib.parse import urljoin
```

Comment:
Use werkzeug.urls, not urllib.parse

Reply:
what's the difference?

Reply:
Mostly that we use werkzeug.urls everywhere. Partially for compat' during the python 3 transition, partially because it often deals better with the bytes/text dichotomy.